src/components/HorizontalStepper/HorizontalStepper.tsx

```diff
@@ -0,0 +1,73 @@
+import clsx from 'clsx';
+import React from 'react';
+import styles from './HorizontalStepper.module.css';
+import HorizontalStep from '../HorizontalStep';
+
+export interface Props {
+  /**
+   * Identifies which index is the active step.
+   */
+  activeIndex: number;
```

Comment:
Should we add a check to make sure that the `activeIndex` is within the length of `steps`? Ex: if there are 3 steps and they pass in `actvieIndex={3}`, throw a console warning.

Reply:
Maybe also mention in the prop comment that it's using a 0-based index just in case; I get thrown off by that all the time.

Reply:
Good suggestions. For the check, all I know is the `console.warn` method so I can do that (unless other suggestions). Will definitely add that it's a 0-based index, thanks!